recipes/backup_client.rb

```diff
@@ -17,10 +18,27 @@
 template "#{node['stash']['home_path']}/backup-config.properties" do
   source 'backup-config.properties.erb'
   owner node['stash']['user']
-  mode '0644'
-  variables :backup_client => settings['backup_client']
+  mode '0600'
+  variables :backup => settings['backup']
 end
 
 link "#{node['stash']['backup_client']['install_path']}/stash-backup-client/backup-config.properties" do
   to "#{node['stash']['home_path']}/backup-config.properties"
 end
+
+directory node.set['stash']['backup']['backup_path'] do
+  owner node['stash']['user']
+  mode '0750'
```

Comment:
Now I got it. Of course, we should not override permissions to "/tmp". 
So, we could just add a guard to this 'directory' resource:
```ruby
...
not_if { ::Dir.exists?(node['stash']['backup']['backup_path']) }
```